File: fst.kru/src/models/Director.ts
import { Position } from "./Position";

export interface Director {
  id: number;
  fullName: string;
  imageName: string;
  isUsed: boolean;
  description: string;
  hidden: boolean;
  positionId: number;
  position: Position;
}

export interface DirectorPosition {
  id: number;
  positionName: string;
}

File: api/api/Models/director/Director.cs
using api.Models.Person;

namespace api.Models.director
{
    public class Director
    {
        public int ID { get; set; }
        public string fullName { get; set; }
        public string? ImageName { get; set; }

        //ปิด isUsed ด้วย
        //public bool IsUsed { get; set; }
        public string Description { get; set; }
        public bool Hidden { get; set; }
        
        public int PositionId { get; set; }
        public Position Position { get; set; }
    }
}
